lib/automation.py

```diff
@@ -22,19 +23,22 @@
                         FunsizeTaskCompletedQueue,
                         ReleaseTaskCompletedQueue,
                         )
+import lib.tc as tc
+import lib.treeherder as treeherder
 
 
 class FirefoxAutomation:
 
-    def __init__(self, configfile, pulse_authfile, debug, log_folder, logger,
-                 message=None, display_only=False):
+    def __init__(self, configfile, pulse_authfile, treeherder_configfile, debug,
+                 log_folder, logger, message=None, display_only=False):
```

Comment:
I don't like the different config files. I think I will do a refactor of the jenkins and pulse auth ones as a follow-up.